src/logic/operate.js

```diff
@@ -0,0 +1,36 @@
+import Big from 'big.js';
+
+const operate = (numOne, numTwo, operation) => {
+  const first = Big(numOne);
+  const second = Big(numTwo);
+  let result = 0;
+
+  switch (operation) {
+    case '+':
+      result = first.plus(second);
+      break;
+
+    case '-':
+      result = first.minus(second);
+      break;
+
+    case 'x':
+      result = first.times(second);
+      break;
+
+    case '%':
+      result = first.mod(second);
+      break;
+
+    case '/':
+      result = first.div(second);
+      break;
+
+    default:
+      break;
+  }
+  window.console.log(result.toString());
```

Comment:
Let's remove `console.log`
No `console.log` committed to the repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools.

![image](https://user-images.githubusercontent.com/31897434/91425902-a9fbc300-e853-11ea-9018-510c1ca778f2.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/